federation/pkg/kubefed/util/util.go

```diff
@@ -130,6 +132,25 @@ func (a *adminConfig) getClientConfig(context, kubeconfigPath string) clientcmd.
 	return clientcmd.NewNonInteractiveDeferredLoadingClientConfig(&loadingRules, overrides)
 }
 
+type MapValue map[string]string
```

Comment:
Instead of defining a new type, better to reuse the existing ones. https://github.com/kubernetes/apiserver/blob/master/pkg/util/flag/configuration_map.go#L25